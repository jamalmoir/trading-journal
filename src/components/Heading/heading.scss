.Heading {
  font-family: "Open Sans", sans-serif;
  font-size: 1.5em;
  letter-spacing: -0.05em;
  margin-bottom: 1em;
  font-weight: 600;
}