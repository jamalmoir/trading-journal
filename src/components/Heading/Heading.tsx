import * as React from 'react';
import './heading.scss';


interface HeadingProps {
  className?: string;
  text: string;
}

export const Heading = (props: HeadingProps) => {
  return (
    <h1 className={ 'Heading ' + props.className }>{ props.text }</h1>
  )
}