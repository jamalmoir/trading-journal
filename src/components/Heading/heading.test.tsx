import * as React from 'react'
import { Heading } from './Heading'
import { shallow } from 'enzyme'

describe('<Heading />', () => {
  it('matches snapshot', () => {
    const component = shallow(<Heading text="Hello Test" />)
    expect(component).toMatchSnapshot()
  })
})
