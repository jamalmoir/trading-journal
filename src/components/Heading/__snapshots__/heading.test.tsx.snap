// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> matches snapshot 1`] = `
<h1
  className="Heading undefined"
>
  Hello Test
</h1>
`;
